CMakeLists.txt

```diff
@@ -857,7 +857,10 @@ if(EXPERIMENTAL AND ENABLE_TBB)
   target_compile_options(OpenSCAD PRIVATE 
     -DENABLE_TBB
     -DTHRUST_HOST_SYSTEM=THRUST_HOST_SYSTEM_TBB
+    -DTHRUST_DEVICE_SYSTEM=THRUST_DEVICE_SYSTEM_TBB
   )
+  pkg_check_modules(TBB tbb REQUIRED)
+  target_link_libraries(OpenSCAD PRIVATE ${TBB_LIBRARIES})
```

Comment:
You should probably do this instead:
```
find_package(TBB REQUIRED)
target_link_libraries(OpenSCAD PRIVATE TBB::tbb)
```

It sounds like you want to patch Manifold, not OpenSCAD though.  Not sure how good it is to have both Manifold and OpenSCAD link TBB in their own way.

Reply:
I think this is not patching manifold: openscad does use thrust for parallel minkowski. As long as the tbb version is the same, I think linking tbb in both should work fine.

Reply:
Oh, I hadn't realized; minkowski for Manifold does indeed use TBB.